### PR TITLE
Uses more single quotes

### PR DESCRIPTION
**Pull request by foliengriller**
I have tested the install-process once on debian 9 with apache 2.4, without any problems.

### CODE REVIEW

File: install/lib/class.FroxlorInstall.php [hunk 1]
```diff
@@ -53,23 +53,23 @@ class FroxlorInstall
...
 	 * @var array
 	 */
-	private $_data = null;
+	private $_data;
 
 	/**
 	 * supported languages for install
 	 */
-	private $_languages = array(
+	private $_languages = [
```
Comment (d00p) [inline install/lib/class.FroxlorInstall.php]: unnecessary and it's not the froxlor way...we use array() everywhere

File: install/lib/class.FroxlorInstall.php [hunk 2]
```diff
@@ -39,7 +39,7 @@ class FroxlorInstall
 	 *
 	 * @var string
 	 */
-	private $_basepath = null;
+	private $_basepath;
```
Comment (d00p) [inline install/lib/class.FroxlorInstall.php]: why not initialize variables? unnecessary

File: install/lib/class.FroxlorInstall.php [hunk 3]
```diff
@@ -90,8 +90,8 @@ class FroxlorInstall
 	 */
 	public function __construct()
 	{
-		$this->_basepath = dirname(dirname(dirname(__FILE__)));
-		$this->_data = array();
+		$this->_basepath = dirname(dirname( __DIR__ ));
```
Comment (d00p) [inline install/lib/class.FroxlorInstall.php]: yes, makes sense

File: install/lib/class.FroxlorInstall.php [hunk 4]
```diff
@@ -102,7 +102,7 @@ public function run()
 		// send headers
 		$this->_sendHeaders();
 		// check if we have a valid installation already
-		$this->_checkUserdataFile();
+		$this->_checkUserDataFile();
```
Comment (d00p) [inline install/lib/class.FroxlorInstall.php]: unnecessary

File: install/lib/class.FroxlorInstall.php [hunk 5]
```diff
@@ -205,15 +205,15 @@ private function _checkPostData()
...
-		$content = "<table class=\"noborder\">";
+		$content = '<table class="noborder">';
 		
 		// check for mysql-root-connection
 		$content .= $this->_status_message('begin', $this->_lng['install']['testing_mysql']);
 
 		$options = array(
 			'PDO::MYSQL_ATTR_INIT_COMMAND' => 'SET names utf8'
 		);
-		$dsn = "mysql:host=" . $this->_data['mysql_host'] . ";";
```
Comment (d00p) [inline install/lib/class.FroxlorInstall.php]: unnecessary

File: install/lib/class.FroxlorInstall.php [hunk 6]
```diff
@@ -222,9 +222,9 @@ private function _doInstall()
 			try {
 				$db_root = new PDO($dsn, $this->_data['mysql_root_user'], '', $options);
 				// set the given password
-				$passwd_stmt = $db_root->prepare("
+				$passwd_stmt = $db_root->prepare( '
```
Comment (d00p) [inline install/lib/class.FroxlorInstall.php]: all queries are enclosed in " "  - and not single quotes

**Comment by d00p** [comment]
Okay, so may unnecessary changes; I think it only makes sense where quotes are quoted in a string, nowhere else

**Comment by d00p** [comment]
wanted to add: i do not try to avoid short array syntax, it's there since php-5.4  but if you start that in one file, you have to update all files thats why i said, better not

**Comment by foliengriller** [comment]
Now I just commited the simple cases. This imho leads to more consistency in the code...

**Comment by d00p** [comment]
Consistency in only one file is meaningless

**Pull request closed without merging**